ui/appui-react/src/appui-react/selection/HideIsolateEmphasizeManager.ts

```diff
@@ -578,6 +580,8 @@ export class HideIsolateEmphasizeManager extends HideIsolateEmphasizeActionHandl
       HideIsolateEmphasizeActionHandler.hideIsolateEmphasizeUiSyncId
     );
 
+    if (!clearSelection) return;
```

Comment:
Please add a test that cover this option.

Reply:
I don't see any tests that cover any part of HideIsolateEmphasizeManager, only ui\appui-react\src\test\selection\HideIsolateEmphasizeActionHandler.test.tsx that fakes manager. Do you want to cover this param in fake manager tests? Or am I missing something?

Reply:
You are right, for some reason, this whole class is istanbul ignored, that shouldnt be the case but that's not on you to add these tests!